TheHuntIsOn.cs

```diff
@@ -74,8 +75,7 @@ public TheHuntIsOn()
 
     public override void Initialize(Dictionary<string, Dictionary<string, GameObject>> preloadedObjects)
     {
-        if (ModHooks.GetMod("HKMP", true) is Mod)
-            SetupHKMP();
+        SetupHKMP();
```

Comment:
Do the changes require a hard dependency to work? I'd like to keep it as optional if possible.

Reply:
Yes, the mod depends on HKMP. Why would you like to keep it as optional? The mod is meant for manhunts/SvH, which wouldn't be played without HKMP anyways.